apollo-router/src/query_planner/bridge_query_planner.rs

```diff
@@ -361,23 +358,22 @@ impl Service<QueryPlannerRequest> for BridgeQueryPlanner {
                     "missing input file for query".to_string(),
                 )))?;
 
-            let added_labels = match add_defer_labels(file_id, &compiler_guard) {
+            match add_defer_labels(file_id, &compiler_guard) {
```

Comment:
now that add_defer_labels is used at the caching query planner level, this should be removed